code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1361,7 +1361,7 @@
 				M.Knockdown(20, 1, 0) // you should be in a bad spot at this point unless epipen has been used
 		if(81)
 			to_chat(M, "You're knocked out from the exhaustion!") // at this point you will eventually die unless you get charcoal
-			M.remove_trait(TRAIT_SLEEPIMMUNE)
+			M.remove_trait(TRAIT_SLEEPIMMUNE) //Supposed to remove all sleep immune traits.
```

Comment:
I don't really approve, you might gain the trait from a source that shouldn't be removed.

Reply:
Oh shit you're right @XDTM I should just use the new ignore_sleepimmune on the Sleeping function..